.github/workflows/kind-e2e-tests.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
            "${{ secrets.TEST_GITHUB_SECOND_TOKEN }}" \
            "${{ secrets.GITLAB_TOKEN }}"
 
-      - name: Run E2E Tests on pull_request
+      - name: Run E2E Tests on nightly
```

Comment:
"on schedule" makes more sense to me as the cron configuration could change, but if you prefer"on nightly", so be it.